app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <string name="utils_converted_amount">≈ %1$s</string>
+  <string name="utils_pretty_amount"><![CDATA[%1$s%2$s<small>%3$s<small>]]></string>
+  <string name="utils_pretty_amount_with_unit"><![CDATA[%1$s%2$s<small>%3$s<small> %4$s]]></string>
+
+  <string name="utils_ok">Ok</string>
+  <string name="utils_proceed">Pokračovat</string>
+  <string name="utils_cancel">Zrušit</string>
+  <string name="utils_date_just_now">Právě teď</string>
+  <string name="utils_unknown">N/A</string>
+  <string name="utils_copied">Zkopírováno do schránky!</string>
+  <string name="btn_back">Zpět</string>
+  <string name="btn_ok">OK</string>
+  <string name="btn_confirm">Potvrdit</string>
+  <string name="btn_cancel">Zrušit</string>
+  <string name="btn_close">Zavřít</string>
+
+  <!-- //////////////// settings //////////////// -->
+
+  <string name="settings_general_title">Obecné</string>
+  <string name="settings_display_prefs">Zobrazení</string>
+  <string name="settings_security_title">Bezpečnost</string>
+  <string name="settings_display_seed">Obnovovací fráze</string>
+  <string name="settings_seed_security">Nastavení přístupu</string>
+  <string name="settings_advanced_title">Pokročilé</string>
+  <string name="settings_fees">Nastavení poplatků</string>
+  <string name="settings_list_channels">Seznam kanálů</string>
+  <string name="settings_logs">Logy</string>
+  <string name="settings_electrum">Electrum server</string>
+  <string name="settings_mutual_close">Vynucené uzavření kanálů</string>
+  <string name="settings_force_close">Nebezpečná zóna</string>
+
+  <!-- //////////////// notification - watcher //////////////// -->
+
+  <string name="notif_watcher_cheating_title">Prosím zapněte Phoenix</string>
+  <string name="notif_watcher_cheating_message">Byl zjištěn pokus o podvod.</string>
+
+  <string name="notif_watcher_connection_title">Phoenix potřebuje přístup k internetu</string>
+  <string name="notif_watcher_connection_message">Sledování na pozadí potřebuje přístup k internetu.</string>
+
+  <!-- //////////////// close all channels //////////////// -->
+
+  <string name="closechannels_title">Uzavřít kanály</string>
+  <string name="closechannels_checking_channels">Kontroluji kanály, prosím vyčkejte…</string>
+  <string name="closechannels_channels_recap">Právě teď máte %1$d Lightning kanálů s agregovaným zůstatkem %2$s.</string>
+  <string name="closechannels_channels_none">Právě nemáte žádné kanály, které by mohly být vyprázdněny.\n\nJdete do \'Seznam kanálů\' k zobrazení stavu vašich kanálů.</string>
+  <string name="closechannels_mutual_title">Vyprázdnit peněženku</string>
+  <string name="closechannels_mutual_instructions"><![CDATA[
+    Použijte tuto stránku k vyprázdnění vaší peněženky na Bitcoinovou adresu.
+  ]]></string>
+  <string name="closechannels_mutual_input_instructions">Finance nelze odeslat do bitcoinové peněženky. Před odesláním se ujistěte se, že je adresa správná.</string>
```

Comment:
Finance mohou být odeslány do bitcoinové peněženky.

Original: "Funds can be sent to a Bitcoin wallet."